content/handbook/engineering/architecture/design-documents/organization/_index.md

```
@@ -11,142 +11,114 @@ group: Organizations
 
-- Organizationはほぼすべてのの GitLab機能をラップします。
-- Organizations間でデータの読み書きは行えません。詳細は[Organization Isolation](isolation.md)を参照してください。
-- 多くの製品機能は変わりませんが、ほとんどのインスタンスレベル機能は Organization レベルに移動します。レベル変更については[以下](#level-structure)で詳しく説明します。
-- Usersは単一のOrganizationのMemberにのみなれます。
-- Organizationのオーナーになることも、標準メンバーになることもできます。
-- 将来、UsersがMemberになれるOrganizationsの数を複数に拡張することを検討します。
-- Organizationオーナーは、ユーザーアカウントを削除できる能力など、Organization内で管理者スタイルの権限を持ちます。詳細は[以下](#roles-and-permissions)を参照してください。
-- これらの変更はGitLab.com、Self Managed、Dedicatedを含むすべてのGitLabプラットフォームで行われます。
+- Organization は GitLab のほぼすべての機能をラップします。
+- Organization 間でデータを読み書きすることはできません。詳細は [Organization の隔離](isolation.md) を参照してください。
+- 多くのプロダクト機能は変更されませんが、ほとんどのインスタンスレベルの機能は下位に移動し、他の機能は Organization レベルに上位に移動します。レベルの変更については [下記](#level-structure) で詳述されています。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)